GNUmakefile

```diff
@@ -171,7 +171,7 @@ clean:
 	rmdir $(js_output_dir) || true
 
 snowball: $(COMPILER_OBJECTS)
-	$(CC) -o $@ $^
+	$(CC) $(CFLAGS) $(CPPFLAGS) -o $@ $^
```

Comment:
Thanks for the patch.

I agree `$(CFLAGS)` should be passed here, but `$(CPPFLAGS)` shouldn't AFAICS.  The C preprocessor isn't being invoked here.

I checked automake and that passes `$(CFLAGS)` but not `$(CPPFLAGS)` when linking.